### DISCUSSION TITLE
Duplicate Handling

### DISCUSSION THREAD

**Discussion by mitchelbone** [mealie-recipes/mealie#4275] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Prevent / handle differently importing duplicate recipes 

### Please provide a concise description of the feature that would resolve your issue.

I would love to see Mealie either just not import duplicate recipes (based on URL) or somehow sandbox them for review before making them live.  I think the only way they are tagged now is a (#) after the name and that seems to be based on the name, not the URL.

### Please consider and list out some caveats or tradeoffs made in your design decision

Low priority.  I doubt there are a ton of users that import as many recipes as I do... don't ask me why.
Thank you for all your hard work.  We love Mealie.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [X] I'm willing to sponsor/pay a developer to do this work

**Comment by ados8** [comment]
Absolutely need this, I'm shocked it doesn't have this function.
I'm currently using Tandoor and I like Mealie more in design and accuracy but this prevents me from switching.
Users are going to import more than once either through forgetting or not knowing what others have imported.
